### PR TITLE
feat: storing theme in local storage

### PR DESCRIPTION
**Pull request by yogeshhrathod**
saving the theme user already selected so next time that theme will be appeared

### CODE REVIEW

File: src/containers/Main.js [hunk 1]
```diff
@@ -28,11 +27,17 @@ export default class Main extends Component {
   }
 
   componentDidMount() {
-    const darkPref = window.matchMedia("(prefers-color-scheme: dark)");
-    this.setState({ isDark: darkPref.matches });
+    if (localStorage.getItem('isDark')==null) {
```
Comment (saadpasta) [inline src/containers/Main.js:30]: Please use strict equality comparison (===) with indentation.

Comment (yogeshhrathod) [reply src/containers/Main.js]: I have fixed this ✔
Thanks

**Review by saadpasta — COMMENTED** [review]
Please remove `README.md` file.

**Comment by kartikcho** [comment]
@saadpasta do we need this? There was another PR I think that tried to add this feature but we rejected it. 
I think that no one will come to the same portfolio more than once in a long time, so leaving cache just to save a button click to set the theme in someone's browser is a very rare use case.
I don't want the work that went into this PR to go to waste but then this wasn't a planned issue afaik. Open to hearing your thoughts but I don't think we need this.

**Comment by yogeshhrathod** [comment]
You are right @kartikcho, but if somebody checking out the portfolio she/she will not know the you motto of saving cache. and without this feature its looks buggy, people will not revisit the page but they can refresh that page and there the can think that is as misbehavior
Thanks

**Comment by kartikcho** [comment]
That is a fair argument @yogeshhrathod and could be useful. But personally I don't think this is necessary.

This isn't a big change so I'm not feeling strongly against this, I guess you can take the final decision @saadpasta

**Comment by saadpasta** [comment]
@kartikcho I think it's a small feature and won't impact much and it's good to have.

**Review by saadpasta — APPROVED** [review]

**Pull request merged**